### ISSUE TITLE
Error being thrown on v2.26.0: TypeError: Cannot read properties of undefined (reading '_makeRequest')

### ISSUE THREAD

**Issue by pbrennand-francis** [rollbar/rollbar.js#1080]
The latest version (2.26.0) of rollbar seems to be routinely throwing an error for each error tracked:
`TypeError: Cannot read properties of undefined (reading '_makeRequest')`

Strangely, these errors themselves end up in Rollbar. This is in an Angular14 app and I've checked that this doesn't happen for version 2.25.2.

Here's one instance of what is logged to the console:
```
2023-01-03T22:42:08.433Z ERROR [main.9a2577619a9f3692.js:97] TypeError: Cannot read properties of undefined (reading '_makeRequest')
    at rollbar.umd.min.js:1:37555
    at v.invoke (zone.js:372:26)
    at M.run (zone.js:134:43)
    at E._makeZoneRequest (rollbar.umd.min.js:1:37534)
    at E.post (rollbar.umd.min.js:1:37081)
    at rollbar.umd.min.js:1:32666
    at L.<computed> (zone.js:2405:41)
    at v.invokeTask (zone.js:406:31)
    at Object.onInvokeTask (core.mjs:26261:33)
    at v.invokeTask (zone.js:405:60) 
```

Please let me know if I can provide any more information.

**Comment by waltjones** [comment]
@pbrennand-francis Thank you for the report. I'll try to reproduce here.

**Comment by waltjones** [comment]
Fix is released in v2.26.1. Thank you again for the report.

**Comment by pbrennand-francis** [comment]
> Fix is released in v2.26.1. Thank you again for the report.

Thank you for such a quick fix!
